versioned_docs/version-0.26.0/learn-topology.md

```diff
@@ -14,7 +14,7 @@ Note that both types of workers as well as external clients are roles and can be
 
 ![Temporal Overview](/img/docs/learn-topology-overview.png)
 
-At the core of Temporal is a highly scalable multitentant service. The service exposes all its functionality through a strongly typed [Proto API](https://github.com/temporalio/temporal-proto/blob/master/workflowservice/service.proto).
+At the core of Temporal is a highly scalable multitentant service. The service exposes all its functionality through a strongly typed [Proto API](https://github.com/temporalio/temporal-proto/blob/master/temporal/workflowservice/v1/service.proto).
```

Comment:
Looks like that slug changed again. I think this is the new one: https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/service.proto ?